infra/main.tf

```diff
@@ -0,0 +1,67 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.48.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = "eu-central-1"
+  default_tags {
+    tags = {
+      Owner = "Jovan Drobnjak"
+    }
+  }
+  access_key = var.aws_access_key
```

Comment:
We don't want to attach credentials to code as it is a bad practice this code will end up in git.

You should on your local workstation perform "aws cli sso login" and then export which aws profile to use in the shell where you will run terraform.

Terraform backend should in S3 with dynamoDB table used for locking so multiple users can work at the same time.